reference/felix/configuration.md

```diff
@@ -141,6 +141,7 @@ See the [HOWTO guide]({{ site.baseurl }}/maintenance/enabling-bpf) for step-by s
 | BPFDataIfacePattern                / <br/> FELIX_BPFDATAIFACEPATTERN                  | Controls which interfaces Felix should attach BPF programs to in order to catch traffic to/from the external network.  This needs to match the interfaces that Calico workload traffic flows over as well as any interfaces that handle incoming traffic to NodePorts and services from outside the cluster.  It should not match the workload interfaces (usually named cali...).. | regular expression | `^(en.*|eth.*|tunl0$)` |
 | BPFConnectTimeLoadBalancingEnabled / <br/> FELIX_BPFCONNECTTIMELOADBALANCINGENABLED   | Controls whether Felix installs the connect-time load balancer.  In the current release, the connect-time load balancer is required for the host to reach kubernetes services. | true,false |  true |
 | BPFExternalServiceMode             / <br/> FELIX_BPFEXTERNALSERVICEMODE               | Controls how traffic from outside the cluster to NodePorts and ClusterIPs is handled.  In Tunnel mode, packet is tunneled from the ingress host to the host with the backing pod and back again.  In DSR mode, traffic is tunneled to the host with the backing pod and then returned directly; this requires a network that allows direct return. | Tunnel,DSR |  Tunnel |
+| BPFExtToServiceConnmark           / <br/> FELIX_BPFEXTTOSERVICECONNMARK               | Controls a 32bit mark that is set on connections from an external client to a local service. This mark allows us to control how packets of that connection are routed within the host and how is routing intepreted by RPF check. | int | 0x80 |
```

Comment:
I don't think this defaults to `0x80`?  I think it defaults to 0.

Reply:
doh, yeah, EKS is all I have on my mind :sweat_smile: